indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -551,46 +548,123 @@ type("TimetableBlockWholeDayBase", ["TimetableBlockBase"],
      }
    );
 
+function drawBalloon(self, evt, editable) {
+    var timetableBlock = self.div.dom;
+    var entryId = self.eventData.scheduleEntryId ? self.eventData.scheduleEntryId : self.eventData.id.substring(1);
+    var params = {
+       'confId': self.eventData.conferenceId[0],
+       'timetable_entry_id': entryId,
+       'editable': editable ? 1 : 0
+    };
+    var balloonId = '#qtip-' + entryId.toString();
```

Comment:
I don't think `toString()` is needed. Concatenating strings an numbers works fine in JS